spark3/src/test/java/org/apache/iceberg/spark/sql/TestNamespaceSQL.java

```diff
@@ -56,6 +56,8 @@ public void cleanNamespaces() {
 
   @Test
   public void testCreateNamespace() {
+    // Nessie namespaces are explicit and do not need to be explicitly managed
+    Assume.assumeFalse(catalogName.endsWith("testnessie"));
```

Comment:
The concept of a namespace is implicit in Nessie and are therefore not managed through the normal `SupportsNamespaces` interface. We skip tests of this interface when the catalog is a `NessieCatalog`.

Reply:
There are a lot of tests that need this. Should we separate the test cases into different suites?